package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap --hoist",
+    "install": "npm run update:core && lerna bootstrap --hoist",
```

Comment:
Yeah, what do you think the best approach to this is? Add a pre-install step with these node dependencies, or something else?